### PR TITLE
blog: Add support by blabber.im

### PR DESCRIPTION
**Pull request by melvo**

### CODE REVIEW

File: content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: XMPP Providers in blabber.im
+date: 2022-08-17
+---
+
+## Easy Onboarding with Android Chat App
+
+A new version of the Android XMPP chat app blabber.im has been released.
+It provides an easy onboarding.
+Passwords are generated automatically and XMPP providers are suggested.
+That suggestion is based on our curated list of XMPP providers.
```
Comment (wurstsalat3000) [inline content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md:11]:
```suggestion
Suggestions are based on our curated list of XMPP providers.
```

File: content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+![blabber.im integrating XMPP Providers](/static/posts/blabber-im-with-xmpp-providers.png)
+
+## Usage of XMPP Providers' Data during Regular Usage
+
+blabber.im went even a step further.
```
Comment (wurstsalat3000) [inline content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md:17]:
```suggestion
blabber.im event went a step further.
```

File: content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Users can open the page for resetting their passwords on their providers' websites.
```
Comment (wurstsalat3000) [inline content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md:18]:
```suggestion
If users need to reset their password, they are now directly navigated to their providers' password reset web page.
```

File: content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Easy Onboarding with Android Chat App
+
+A new version of the Android XMPP chat app blabber.im has been released.
+It provides an easy onboarding.
+Passwords are generated automatically and XMPP providers are suggested.
+That suggestion is based on our curated list of XMPP providers.
+
+![blabber.im integrating XMPP Providers](/static/posts/blabber-im-with-xmpp-providers.png)
```
Comment (wurstsalat3000) [inline content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md:13]:
```suggestion
![blabber.im integrating XMPP Providers](/posts/blabber-im-with-xmpp-providers.png)
```

**Comment by wurstsalat3000** [comment]
The following files have no copyright and licensing information:

* static/blog/blabber-im-with-xmpp-providers.png
* static/blog/password-reset.png

File: content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md (hunk 1 above)
Comment (melvo) [reply content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md:11]: What about?
```suggestion
Those suggestions are based on our curated list of XMPP providers.
```

File: content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md (hunk 2 above)
Comment (melvo) [reply content/blog/2022-08-17-xmpp-proivders-in-blabber-im.md:17]:
```suggestion
blabber.im even went a step further.
```

**Comment by wurstsalat3000** [comment]
The .md file has a typo "proivders"

**Pull request merged**